charts/rasa-x/values.yaml

```diff
@@ -64,7 +64,12 @@ rasa:
   rabbitQueue: "rasa_production_events"
   # additionalChannelCredentials which should be used by Rasa to connect to various
   # input channels
-  additionalChannelCredentials: ""
+  additionalChannelCredentials:
```

Comment:
Does it work? I expect that an empty object needs to be specified as 
```
additionalChannelCredentials: {}
```

Reply:
Yep, works, but we can also make it explicit. Probably a good idea 👍🏼